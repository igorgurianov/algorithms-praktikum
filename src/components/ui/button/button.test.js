import React from "react";
import renderer from "react-test-renderer";

import { Button } from "./button";

it("Button component renders", () => {
  const btn = renderer.create(<Button text="123" />).toJSON();
  expect(btn).toMatchSnapshot();
});
