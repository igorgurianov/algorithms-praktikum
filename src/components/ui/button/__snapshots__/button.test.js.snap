// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BUTTON Button disabled 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  disabled={true}
  type="button"
>
  <p
    className="text undefined"
  />
</button>
`;

exports[`BUTTON Button loading 1`] = `
<button
  className="text text_type_button text_color_primary button undefined loader "
  disabled={true}
  type="button"
>
  <img
    alt="Загрузка."
    className="loader_icon"
    src="loader.svg"
  />
</button>
`;

exports[`BUTTON Button w/ text 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  type="button"
>
  <p
    className="text undefined"
  >
    123
  </p>
</button>
`;

exports[`BUTTON Button w/o text 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  type="button"
>
  <p
    className="text undefined"
  />
</button>
`;
